client/dashboard/task-list/tasks/shipping/index.js

```diff
@@ -180,11 +180,17 @@ class Shipping extends Component {
 			{
 				key: 'label_printing',
 				label: __( 'Enable shipping label printing', 'woocommerce-admin' ),
-				description: __(
-					'With WooCommerce Services and Jetpack you can save time at the ' +
-						'Post Office by printing your shipping labels at home',
-					'woocommerce-admin'
-				),
+				description: [ 'GB', 'CA', 'AU' ].includes( countryCode )
+					? __(
+							'We recommend using ShipStation to save time at the post office by printing your shipping ' +
+								'labels at home. Try ShipStation free for 30 days. Learn more.',
```

Comment:
Do we have a learn more link to add here? Or should we drop that text.

Reply:
Updated this to point to `https://docs.woocommerce.com/document/shipstation-for-woocommerce/`.  I think this is consistent with our other links.  @pmcpinto let me know if you think this should change.